visidata/loaders/fec.py

```diff
@@ -230,7 +210,7 @@ class FECFiling(Sheet):
     @asyncthread
     def reload(self):
         from fecfile import fecparser
-        self.rows = []
+        self.rows = []  # rowdef:  Sheet, of a type from COMPONENT_SHEET_CLASSES.values()
```

Comment:
In general if the rowdef is a Sheet then the parent class is IndexSheet, which provides default columns and other functionality.  I'm not sure if it applies here but it might be worth exploring.

Reply:
Good to know! I'll make note to look into that for future work with rows that are Sheets. But since this FEC loader has been broken for a while without any user complaints, I suspect it's not often used, so I don't plan to spend more time on it.